independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/CreateProjectCommandHandler.java

```diff
@@ -244,4 +234,25 @@ private static void addOrigins(final List<ExtensionOrigins> extOrigins, Extensio
             extOrigins.add(eoBuilder.build());
         }
     }
+
+    private void checkMinimumJavaVersion(String javaVersionString, List<Extension> extensions) throws QuarkusCommandException {
+        final List<Extension> incompatibleExtensions = new ArrayList<>();
+        final int javaVersion = javaVersionString == null ? CreateProjectHelper.DEFAULT_JAVA_VERSION
+                : Integer.parseInt(javaVersionString);
+        for (Extension extension : extensions) {
+            Integer extMinJavaVersion = getMinimumJavaVersion(extension);
+            if (extMinJavaVersion != null
+                    && CreateProjectHelper.JAVA_VERSIONS_LTS.contains(extMinJavaVersion)
```

Comment:
What is the reason for checking whether it's an LTS?

Reply:
This was a way to make sure the value is valid and not a mistake, do you see something else than a LTS in there? Also maybe we should enfore extension to be built on LTS no?

Reply:
I don't think we should be enforcing it.

Reply:
Fixed